src/cli/README.md

```diff
@@ -809,6 +810,64 @@ Set the Thread Child Timeout value.
 Done
 ```
 
+### coex
+
+Get the coex status.
```

Comment:
Should we mention that the `OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_ENABLE` is required?